src/lib/components/chat/Messages/Markdown/MarkdownTokens.svelte

```
@@ -193,14 +193,14 @@
 		{@const html = DOMPurify.sanitize(token.text)}
 		{#if html && html.includes('<video')}
 			{@html html}
-		{:else if token.text.includes(`<iframe src="${WEBUI_BASE_URL}/api/v1/files/`)}
+		{:else if token.text.includes(`<iframe src="${WEBUI_URL}/api/v1/files/`)}
 			{@html `${token.text}`}
```

ESLint

[error] `{@html}` can lead to XSS attack.

(svelte/no-at-html-tags)